features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/NewOverview/NewCustomizedStepper.jsx

```diff
@@ -0,0 +1,427 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import Stepper from '@material-ui/core/Stepper';
+import Step from '@material-ui/core/Step';
+import StepLabel from '@material-ui/core/StepLabel';
+import Tooltip from '@material-ui/core/Tooltip';
+import { FormattedMessage } from 'react-intl';
+import LaunchIcon from '@material-ui/icons/Launch';
+import Alert from 'AppComponents/Shared/Alert';
+import Grid from '@material-ui/core/Grid';
+import StepConnector from '@material-ui/core/StepConnector';
+import ApiContext, { useAPI } from 'AppComponents/Apis/Details/components/ApiContext';
+import CheckIcon from '@material-ui/icons/Check';
+import CloseIcon from '@material-ui/icons/Close';
+import { Link } from 'react-router-dom';
+import grey from '@material-ui/core/colors/grey';
+import CircularProgress from '@material-ui/core/CircularProgress';
+import Button from '@material-ui/core/Button';
+import AuthManager from 'AppData/AuthManager';
+import Typography from '@material-ui/core/Typography';
+
+
+const ColorlibConnector = withStyles({
+    alternativeLabel: {
+        top: 22,
+    },
+    active: {
+        '& $line': {
+            backgroundImage:
+                'linear-gradient(to left, #50BCEC 50%, #B1D31E 50%)',
```

Comment:
Are we using this linear grading effect now ?

Reply:
This is only for the connector, I remove it from
![Screenshot from 2021-03-23 11-56-26](https://user-images.githubusercontent.com/8374895/112102815-d780d300-8bce-11eb-9b7c-142c65100b8b.png)
 Circle view